readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://olhapaliukh.github.io/layout_stars/)
+- [TEST REPORT LINK](https://olhapaliukh.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to PR description

Reply:
Still not added
<img width="922" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/6998d2d5-198c-4637-81e2-823b21a2a362">